api/service/projects_service.go

```diff
@@ -250,3 +283,116 @@ func (service *projectsService) filterAuthorizedProjects(ctx context.Context, pr
 	}
 	return authorizedProjects, nil
 }
+
+func (service *projectsService) handleUpdateProjectRequest(project *models.Project) (*models.Project, map[string]interface{}, error) {
+	if service.updateProjectConfig.Endpoint == "" || service.updateProjectConfig.PayloadTemplate == "" ||
+		service.updateProjectConfig.ResponseTemplate == "" {
+		return project, nil, nil
```

Comment:
If the endpoint, or any of the templates, is an empty string (an external endpoint is not configured by the operator of MLP), we should still ensure that `Ok(project)` gets returned to the user right? How does this happen?